modules/installing-ocp-agent.adoc

```diff
@@ -51,38 +51,39 @@ cat << EOF > ./my-cluster/install-config.yaml
 apiVersion: v1
 baseDomain: test.example.com
 compute:
-- architecture: amd64
+- architecture: amd64 <1>
   hyperthreading: Enabled
   name: worker
   replicas: 0
 controlPlane:
-  architecture: amd64
+  architecture: amd64 <1> 
   hyperthreading: Enabled
   name: master
   replicas: 1
 metadata:
-  name: sno-cluster <1>
+  name: sno-cluster <2>
 networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.111.0/16
-  networkType: OVNKubernetes <2>
+  networkType: OVNKubernetes <3>
   serviceNetwork:
   - 172.30.0.0/16
 platform:
   none: {}
-pullSecret: '<pull_secret>' <3>
+pullSecret: '<pull_secret>' <4>
 sshKey: |
-  '<ssh_pub_key>' <4>
+  '<ssh_pub_key>' <5>
   EOF
 ----
 +
-<1> Required.
-<2> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<3> Enter your pull secret.
-<4> Enter your ssh public key.
+<1> System architecture, valid values are `amd64` and `arm64`.
+<2> Required.
+<3> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
+<4> Enter your pull secret.
+<5> Enter your ssh public key.
```

Comment:
I know you're just adding one here and this diff misleadingly makes it look like all of these are new. But one thing you might consider while you're in here :)

Ideally, these should all be complete sentences. When we are telling the user to add a value, using a verb like "specify" is a pretty common form. So something like this might work:

```suggestion
<1> Specify the system architecture. Valid values are `amd64` and `arm64`.
<2> Required.
<3> Specify the cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
<4> Specify your pull secret.
<5> Specify your SSH public key.
```

I'm not sure what callout `2` aside from that it's required. Would be good to describe it if possible.